Gyanthakur/Dev_Profiles
# Improve responsiveness of Contact Us page

## Description

Added new CSS elements to tackle the responsiveness of the Contact Us page. Additionally, corrected spelling mistakes in the navbar (Contact Us was displayed as Content Us).

Fixes: #279 

Use [x] to represent a checked (ticked) box.✅
Use [ ] to represent an unchecked box.❌

## Type of Change

- [ ❌] Profile Added
- [❌ ] Project Added
- [✅ ] Bug fix
- [ ❌] New feature
- [ ❌] Documentation update
- [❌ ] Other (please specify):

## How to Test

1. Go to the Contact Us page
2. By using web dev tools change the screen size
3. Check for responsiveness at different screen sizes

## Checklist

- [✅ ] I have performed a self-review of my code.
- [ ✅] I have commented my code, particularly in hard-to-understand areas.
- [ ✅] I have made corresponding changes to the documentation (if applicable).
- [ ✅] My changes generate no new warnings.
- [✅ ] I have added tests to cover my changes (if applicable).
- [ ✅] All new and existing tests pass.
